### PR TITLE
Update gems and clean up

### PR DESCRIPTION
**Pull request by jornvandebeek**

### CODE REVIEW

**Comment by Hampei** [comment]
It is customary to change the version in a separate commit and not in a pull-request.

Just removing support for some ruby version is not a reason to move to the next mayor version, though it is probably time to move this thing to 1.0.

Document the required ruby_version in the gemspec, so people don't upgrade by accident.

`s.required_ruby_version = '>= 2.2.0'`

File: .gitignore [hunk 1]
```diff
@@ -14,39 +10,3 @@ doc
 
 # jeweler generated
```
Comment (timraasveld) [inline .gitignore]: This can be removed if the gem is no longer included

**Comment by hampei** [comment]
Also adding the version in code is always a good idea. You

**Review by timraasveld — APPROVED** [review]
LGTM

**Review by Hampei — APPROVED** [review]
LGTM

**Pull request merged**